addons/sys_rack/XEH_postInit.sqf

```diff
@@ -120,7 +120,7 @@ if (vehicle acre_player != acre_player) then {
         false
     };
 
-    if ([_rackId] call FUNC(getMountedRackRadio) != "") exitWith {
+    if ([_rackId] call EFUNC(api,getMountedRackRadio) != "") exitWith {
```

Comment:
API function just calls `FUNC(getMountedRadio)` if all tests pass, so this should probably be:
```suggestion
    if ([_rackId] call FUNC(getMountedRadio) != "") exitWith {
```
Ideally components should also never call API component functions, only the other way around.